scripts/benchmark_dashboard.py

```
@@ -0,0 +1,74 @@
+
+
+def benchmark(iterations=20):
+    client = app.test_client()
+
+    print(f"Benchmarking dashboard with {iterations} iterations...")
+
+    # Warm up
+    client.get("/")
+
+    start_time = time.time()
+    for i in range(iterations):
```

Ruff (0.14.14)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)